homeassistant/components/deconz/deconz_event.py

```diff
@@ -47,6 +47,8 @@ def async_update_callback(self, force_update=False):
                 CONF_UNIQUE_ID: self.serial,
                 CONF_EVENT: self._device.state,
             }
+            if hasattr(self._device, CONF_GESTURE) and self._device.gesture is not None:
```

Comment:
All events are of type zhaswitch which now has the gesture attribute.
```suggestion
            if self._device.gesture:
```